platform/android/app/src/main/kotlin/coop/polypoly/polypod/settings/MainFragment.kt

```diff
@@ -15,6 +18,18 @@ class MainFragment : PreferenceFragmentCompat() {
         setPreferencesFromResource(R.xml.settings, rootKey)
         findPreference<Preference>("version")?.summary = RuntimeInfo.VERSION
 
+        findPreference<Preference>("biometricEnabledKey")
```

Comment:
Can you consider using a constant for this string literal, which is repeated twice?

Reply:
for a matter of consistency with rest preferences, I left it as a string (would require a separate xml setting for reading these values maybe if I got it correctly). 
We could do have a var here as it's used twice, matter of tastes though